### PR TITLE
Add css Selectors

### PR DESCRIPTION
**Pull request by Antonieta13**

### CODE REVIEW

File: src/test/java/BuyShirt.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+            new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".cart_navigation span"))).click();
+
+            new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".checker"))).click();
+
+            new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".cart_navigation span"))).click();
+
+            new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector("a.bankwire"))).click();
+
+            new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".cart_navigation span"))).click();
```
Comment (aperdomob) [inline src/test/java/BuyShirt.java]: fix the indentation

File: src/test/java/BuyShirt.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+
+          // driver.quit();
```
Comment (aperdomob) [inline src/test/java/BuyShirt.java]: why did you comment on this line?

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: src/test/java/BuyShirt.java (hunk 1 above)
Comment (Antonieta13) [reply src/test/java/BuyShirt.java]: Done

File: src/test/java/BuyShirt.java (hunk 2 above)
Comment (Antonieta13) [reply src/test/java/BuyShirt.java]: Just for watch with details the test in the end. I already uncomment

File: src/test/java/BuyShirt.java (hunk 1 above)
Comment (yfarfan) [reply src/test/java/BuyShirt.java]: **First question;** It is possible to have only one `WebDriverWait` for all actions?  So you don't have to instantiate that class every time?

Comment (yfarfan) [reply src/test/java/BuyShirt.java]: **Second question:** `new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable...`  can we do this only for 1 element on the page (the slowest one or something like that) so you don't have to put the wait for all elements?

Comment (aperdomob) [reply src/test/java/BuyShirt.java]: Ambas propuestas son muy vailidas Yeli, sin embargo el objetivo de esta tarea pienso que es que pudiera automatizar un caso, no necesariamente con practicas

Creo que la proxima tarea debería tener en cuenta estas 2 cosas que dices

File: src/test/java/BuyShirt.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+        new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".checker"))).click();
+
+        new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".cart_navigation span"))).click();
+
+        new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector("a.bankwire"))).click();
+
+        new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".cart_navigation span"))).click();
+
+        driver.quit();
```
Comment (aperdomob) [inline src/test/java/BuyShirt.java:75]: add this line in an after method

**Review by aperdomob — CHANGES_REQUESTED** [review]

**Review by aperdomob — APPROVED** [review]

**Pull request merged**